javascript/MLE-Demo/README.md

```diff
@@ -0,0 +1,10 @@
+# Bookstore App in APEX with JavaScript/MLE and GraphQL
+
+## Introduction
+
+This repository contains the source code and documentation for the blog post titled "Building an end-to-end Bookstore App in APEX using JavaScript/MLE and GraphQL," which will be published soon. The blog post will provide a detailed walkthrough of the steps involved in creating a comprehensive Bookstore App using Oracle Application Express (APEX), JavaScript/MLE, and GraphQL.
```

Comment:
I added a placeholder for the upcoming blog post link.

Reply:
Great, thanks! Should we hold back the merge until the blog post goes live or can we coordinate the two events so that you don't have to go back and update it again?

Reply:
The plan is to create another small PR on GitHub where I add the public link to the post to the README.md file once the blog get the final OK for publishing it